website/docs/d/pbr_l1_l2_destination.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "L4-L7 Services"
+layout: "aci"
+page_title: "ACI: aci_pbr_l1_l2_destination"
+sidebar_current: "docs-aci-data-source-pbr_l1_l2_destination"
+description: |-
+  Data source for ACI Destination of L1/L2 Redirected Traffic
+---
+
+# aci_pbr_l1_l2_destination #
+
+Data source for ACI Destination of L1/L2 Redirected Traffic
+
+
+## API Information ##
+
+* `Class` - vnsL1L2RedirectDest
+* `Distinguished Name` - uni/tn-{tenant_name}/svcCont/svcRedirectPol-{pbr_name}/L1L2RedirectDest-{destName}
+* `Distinguished Name` - uni/tn-{tenant_name}/svcCont/backupPol-{pbr_name}/L1L2RedirectDest-{destName}
+
+## GUI Information ##
+
+* `Location` - Tenants -> Policies -> Protocol -> L4-L7 Policy-Based Redirect -> L1/L2 Destinations
+* `Location` - Tenants -> Policies -> Protocol -> L4-L7 Policy-Based Redirect Backup -> L1/L2 Destinations
+
+## Example Usage ##
+
+```hcl
+
+# L4-L7 Policy-Based Redirect
+data "aci_pbr_l1_l2_destination" "l1_l2_destination" {
+  policy_based_redirect_dn = aci_service_redirect_policy.pbr.id
+  dest_name                = "l1_l2_destination"
+}
+
+# L4-L7 Policy-Based Redirect Backup
+data "aci_pbr_l1_l2_destination" "l1_l2_destination" {
+  policy_based_redirect_dn = aci_service_redirect_backup_policy.pbr_backup.id
+  dest_name                = "l1_l2_destination"
+}
+
+```
+
+## Argument Reference ##
+
+* `policy_based_redirect_dn` - (Required) Distinguished name of the parent Policy-Based Redirect object.
```

Comment:
Distinguished name of the parent Policy-Based (Redirect or Redirect Backup) object